cypress/integration/sidebar.spec.js

```diff
@@ -91,17 +92,63 @@ describe('Open the sidebar', function() {
 
 	})
 
+	it('Change to next image with sidebar open', function() {
```

Comment:
Add one screenshot before too so we see the first opened sidebar :)

Reply:
:+1: